### DISCUSSION TITLE
Create-share-link with option to indicate permissions: read only or edit

### DISCUSSION THREAD

**Discussion by jrpelegrina** [abraunegg/onedrive#2108] [General]
Good morning
I'm trying the --create-share-link option which works really well.
Regarding this option, would it be feasible to add the possibility of indicating if the link to be created is to be edited or read-only?

A greeting and congratulations for the work you are doing with this client

**Comment by abraunegg** [comment]
@jrpelegrina 
Please submit a feature request for this to be looked at further.
